Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/xunit.runner.json

```diff
@@ -1,4 +1,5 @@
 {
     "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
-    "parallelizeTestCollections": true
+    "parallelizeTestCollections": true,
+    "maxParallelThreads":  16
```

Comment:
Without this, the execution of integration tests in Visual Studio fails (for me at least locally)

Reply:
Also not with this change? When I select all the integration tests (on the emulator) with this configuration enabled, it does work. Before I added this configuration, it would just keep on spinning without actually running any tests.